### PR TITLE
Add CompareImages and TransformScreenshot Method in TizenDriver

### PR DESCRIPTION
**Pull request by flyofsky**
## Change list

 - Add CompareImages method
 - Add TransformScreenshot method
 - Update summary
 
## Types of changes
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs [hunk 1]
```diff
@@ -64,24 +86,88 @@ public TizenDriver(Uri remoteAddress, DesiredCapabilities desiredCapabilities)
...
+                    }
+                }
+            }
+
+            return true;
+        }
+
+        public Image TransformScreenshot(Screenshot screenshot, Rectangle rectangle)
+        {
+            if (screenshot == null || rectangle == null)
```
Comment (rookiejava) [inline appium-dotnet-driver/Appium/Tizen/TizenDriver.cs]: I think rectangle is safe. Bitmap.Clone() will do instead.

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs [hunk 2]
```diff
@@ -64,24 +86,88 @@ public TizenDriver(Uri remoteAddress, DesiredCapabilities desiredCapabilities)
...
         public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabilities, TimeSpan commandTimeout)
             : base(service, SetPlatformToCapabilities(desiredCapabilities, Platform), commandTimeout)
         {
         }
 
+        public bool CompareImages(Image firstImage, Image secondImage)
+        {
+            if (firstImage == null || secondImage == null)
+            {
+                throw new NullReferenceException("Screenshot and Rectangle should not be null");
```
Comment (rookiejava) [inline appium-dotnet-driver/Appium/Tizen/TizenDriver.cs]: message should be fixed correctly.

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs [hunk 3]
```diff
@@ -64,24 +86,88 @@ public TizenDriver(Uri remoteAddress, DesiredCapabilities desiredCapabilities)
...
+            }
+
+            var firstBitmap = new Bitmap(firstImage);
+            var secondBitmap = new Bitmap(secondImage);
+            Color firstPixel;
+            Color secondPixel;
+            int firstGrayScale = 0;
+            int secondGrayScale = 0;
+
+            for (int i = 0; i < firstImage.Width; i++)
```
Comment (rookiejava) [inline appium-dotnet-driver/Appium/Tizen/TizenDriver.cs]: where comes this approach?  just of quick scan, it seems it has poor performance.

Could you optiming it again? 

For example, we can check length/size of both images before looping. Or we can consider to use `LockBitmap` ,.. and so on.

**Review by rookiejava — CHANGES_REQUESTED** [review]

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs (hunk 3 above)
Comment (rookiejava) [reply appium-dotnet-driver/Appium/Tizen/TizenDriver.cs]: Performance comparison between LockBits and GetPixel.

https://mfranc.com/programming/operacje-na-bitmapkach-net-1/

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs (hunk 2 above)
Comment (flyofsky) [reply appium-dotnet-driver/Appium/Tizen/TizenDriver.cs]: Done

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs (hunk 3 above)
Comment (flyofsky) [reply appium-dotnet-driver/Appium/Tizen/TizenDriver.cs]: Done

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs (hunk 1 above)
Comment (flyofsky) [reply appium-dotnet-driver/Appium/Tizen/TizenDriver.cs]: Done

**Review by rookiejava — APPROVED** [review]

**Pull request merged**